src/services/apes/apes.ts

```diff
@@ -12,12 +12,12 @@ import {
 } from "types/aws";
 import { ChainID } from "types/chain";
 import { DetailedCurrency } from "types/components";
+import { DonationSource } from "types/lists";
 import { apiEnv } from "../constants";
 import { version as v } from "../helpers";
 import { tags } from "./tags";
 
-type StripePaymentIntentParams = {
-  type: "one-time" | "subscription";
+type FiatDonation = {
```

Comment:
different fiat donation types share the same base type in server so maintanable to also extract this base in web-app